src/ui/Facet/Facet.ts

```diff
@@ -165,15 +165,16 @@ export class Facet extends Component {
      * If you have two facets with the same field on the same page, you should specify an `id` value for at least one of
```

Comment:
Yes, I think it's better to take the safest approach of simply not accepting something that does not go in an URL, versus trying to work around it, because in my experience this never works how we expect it to work. An URL is a very complicated thing when you start going into corner cases.